pom.xml

```diff
@@ -96,6 +96,7 @@
     <maven.build-helper.plugin.version>1.12</maven.build-helper.plugin.version>
     <maven.eclipse.plugin.version>2.10</maven.eclipse.plugin.version>
     <maven.exec.plugin.version>1.6.0</maven.exec.plugin.version>
+    <maven.setversion.plugin.version>2.13.0</maven.setversion.plugin.version>
```

Comment:
it we want to stick to the maven plugin name, it's:
```
<maven.versions.plugin.version>
```

Reply:
Good catch, I will perform the rename during the final commit!